pom.xml

```diff
@@ -9,24 +9,22 @@
   
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.bdgenomics.adam</groupId>
-  <artifactId>adam-parent</artifactId>
+  <artifactId>adam-parent_2.10</artifactId>
   <version>0.16.1-SNAPSHOT</version>
   <packaging>pom</packaging>
-  <name>ADAM</name>
+  <name>ADAM_2.10</name>
   <description>A fast, scalable genome analysis system</description>
   <url>http://bdgenomics.org/</url>
 
   <properties>
     <java.version>1.7</java.version>
-    <scala.version>2.10.4</scala.version>
```

Comment:
why take these out, OOC? just feel like it's cleaner to not interpolate these variables below?